### ISSUE TITLE
Improve readability of Station code

### ISSUE THREAD

**Issue by jawaad-ahmad** [brata-hsdc/brata.station#6]
Remove boilerplate comment blocks from methods.  Having no comment is better than having a large comment with no information.  Code readability is improved when you can see more code.

Also, having that many TODO's overloads the TODO-flagging mechanism in Eclipse, making it difficult to see actual TODO's that matter.

See:
- https://www.python.org/dev/peps/pep-0257  Docstring Conventions
- https://www.python.org/dev/peps/pep-0008  Style Guide for Python Code

I also would have found it helpful to start out with a high-level architectural diagram showing the different threads and their roles, and a high-level inheritance diagram showing the high-level abstract entities and their concrete instantiations.  Any contextual information that would help me as a developer to make the initial decisions about how and where to make modifications and additions to the station functionality would be a big time-saver and will also lead to a more consistently-structured application with less need for review and oversight.

This continues brata-hsdc/brata#75.

**Comment by DragonmasterJ** [comment]
I'd like to break this up into issues for each separate station type as well as the general station code. This way we can break up the branches for people to work on them. I'll open up those issues soon and update.

**Comment by jawaad-ahmad** [comment]
Do you think we need a dedicated issue for this, or at this point should it
be something to address while we're in the specific area that we're
touching and anything else along the way?

**Comment by ellerychan** [comment]
I suppose it will get taken care of whether we make an issue or not.

**Comment by DragonmasterJ** [comment]
Good point. I was thinking if someone was going back and trying to clean up comments it could get messy without specific branches, but if we just leave it to do it diligently as we are changing code that makes perfect sense as well.
